runtime/gc_modron_startup/mminit.cpp

```diff
@@ -3325,11 +3325,8 @@ initializeIndexableObjectHeaderSizes(J9JavaVM* vm)
 #else /* defined(J9VM_ENV_DATA64) */
 	setIndexableObjectHeaderSizeWithoutDataAddress(vm);
 #endif /* defined(J9VM_ENV_DATA64) */
-	if (MM_GCExtensions::getExtensions(vm)->isVirtualLargeObjectHeapEnabled) {
-		vm->unsafeIndexableHeaderSize = 0;
-	} else {
-		vm->unsafeIndexableHeaderSize = vm->contiguousIndexableHeaderSize;
-	}
+	/* set default unsafeIndexableHeaderSize */
+	vm->unsafeIndexableHeaderSize = vm->contiguousIndexableHeaderSize;
```

Comment:
let's do changes related to setting unsafeIndexableHeaderSize as a separate PR before we proceed with array access macro changes

that looks like something should have been done as the part of original offheap changes (we should not have queried isVirtualLargeObjectHeapEnabled is a relatively common path). the new way is cleaner (set default, and override it for offheap enabled on its own specific path)